app/src/main/res/values/strings.xml

```diff
@@ -224,15 +224,15 @@
     <string name="file_name_empty_error">Filename cannot be empty</string>
     <string name="error_occurred_detail">An error occurred: %1$s</string>
     <string name="no_streams_available_download">No streams available to download</string>
-    <string name="saved_tabs_invalid_json">Using default tabs, error while reading saved tabs</string>
+    <string name="saved_tabs_invalid_json">Could not read saved tabs, so using default ones</string>
     <string name="restore_defaults">Restore defaults</string>
-    <string name="restore_defaults_confirmation">Do you want to restore the defaults?</string>
+    <string name="restore_defaults_confirmation">Do you want to restore defaults?</string>
     <!-- error activity -->
     <string name="sorry_string">Sorry, that should not have happened.</string>
     <string name="guru_meditation" translatable="false">Guru Meditation.</string>
-    <string name="error_report_button_text">Report error via e-mail</string>
+    <string name="error_report_button_text">Report this per e-mail</string>
```

Comment:
I am not sure about this string. What about "Report error by e-mail"?

Reply:
Well, via, by and per are all correct here. Tbh I prefer via (although it is often used formal context).

Reply:
Per is correct, but it is American-ish. So I'd use either by or via

Reply:
I suggest "Report this error via e-mail". It is the most descriptive and straightforward.